src/features/game/types/images.ts

```diff
@@ -4315,4 +4315,8 @@ export const ITEM_DETAILS: Items = {
     image: kingOfBears,
     description: "?",
   },
+  Wagon: {
+    image: Wagon,
+    description: "?",
```

Comment:
Wagon might need a description! Add whatever feels right 👍 . Bonus points for adding a description to the king of bears 😆 

![1](https://github.com/user-attachments/assets/129604ce-750d-4ec4-806c-b105441c3f43)